docs/source/dynamic_typing.rst

```diff
@@ -80,7 +80,7 @@ operations:
         n = 1  # type: int
         n = o         # Error!
 
-You can use ``cast()`` (see chapter :ref:`casts`) or ``isinstance`` to
-go from a general type such as ``object`` to a more specific
-type (subtype) such as ``int``.  ``cast()`` is not needed with
+You can use :py:func:`~typing.cast` (see chapter :ref:`casts`) or :py:func:`isinstance` to
```

Comment:
I have altered the rendering by including the parentheses to `isinstance`. Before:

![image](https://user-images.githubusercontent.com/4455652/66438953-768f2a80-ea2e-11e9-892e-5fab1571a84c.png)

After:

![image](https://user-images.githubusercontent.com/4455652/66438991-8eff4500-ea2e-11e9-83d2-cfc79fcc24c2.png)

Both are functions - shouldn't both have parentheses in this case?

Reply:
+1. The only time it's better not to add `()` to a function is when we're talking about the function object (e.g. in the context of creating an alias or accessing attributes).